arches/app/templates/views/components/plugins/manifest-manager.htm

```diff
@@ -292,11 +292,11 @@ <h2>
         <div style="display: flex; padding: 15px 25px; flex-direction: column; justify-content: center; align-items: center;">
         <div style="display:flex; flex-direction: column; width: 300px">
             <div class="manifest-editor-input">
-                <input class="form-control input-lg widget-input"
-                    data-bind="value: manifest,
-                    select2Query: {
+                <select class="form-control input-lg widget-input"
+                    data-bind="select2Query: {
```

Comment:
WAVE indicates this is missing a form label

Reply:
@johnatawnclementawn I'm not worried about that so much as making sure the selectwoo stuff all works.  That said if you want to push a quick fix then please do!